.gitignore

```diff
@@ -186,6 +186,7 @@ sdk/template/template-dpg/src/src
 
 # tshy
 .tshy-build-tmp
+.tshy-build
```

Comment:
has it been changed from `.tshy-build-tmp` to `.tshy-build`?

Reply:
looks like yes: https://github.com/isaacs/tshy/commit/b0941304c34d368c4796fd2982b9189b14c094a0